components/WorkerView/WorkerRecap.tsx

```diff
@@ -1,21 +1,23 @@
 import { Worker } from 'types';
 
+import style from './WorkerRecap.module.scss';
+
 const WorkerRecap = ({
   firstName,
   lastName,
   role,
   teams,
 }: Worker): React.ReactElement => {
   return (
-    <div>
-      <h2 className="gov-weight-lighter">
+    <div className={style.detailsContainer}>
+      <h2 className={`lbh-heading-h2 gov-weight-lighter ${style.title}`}>
```

Comment:
`gov-weight-lighter` is no longer needed here - we've consolidated all font weights down to just two, which are correctly applied by default.